services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+
+from db.models import Ticket, User, Order, MovieSession
+from django.db.models import DateTimeField, QuerySet
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: DateTimeField = None
+) -> None:
+    user = User.objects.get(username=username)

```

Comment:
Use `get_user_model()` instead of `User`. Check out the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency).